packages/transition-backend/file/definitions/definitions-en.tex

```diff
@@ -178,118 +185,100 @@ \subsection*{Véhicules, unités et garages • \textit{Vehicles, units and depo
     C{.11\NetTableWidth}%
     C{.2\NetTableWidth}%
     L{.3\NetTableWidth}%
-  }
+}
 \hline
-\makecell[r]{Définition \\ \textit{Definition}} & \makecell[c]{Symbole \\ \textit{Symbol}} & \makecell[c]{Unité \\ \textit{Unit}} & \makecell[c]{Expression \\ \textit{Expression}} & \makecell[l]{Description \\ \textit{Description}} \\ 
+\makecell[r]{Definition} & \makecell[c]{Symbol} & \makecell[c]{Unit} & \makecell[c]{Expression} & \makecell[l]{Description} \\
 \hline
 \hline
 \endhead
 \label{floor_area_per_standee}
-\makecell[r]{Superficie au sol par place debout \\ \textit{Floor area per standee}} & \[e\] & \[m^2\] & - & Superficie prévue pour chaque place debout (empreinte au sol). Confortable, circulation convenable: \(0.25 m^2\); Inconfortable, circulation difficile: \(0.15 m^2\) \\
-\hline
-\label{vehicle_capacity}
-\makecell[r]{Capacité de véhicule \\ \textit{Vehicle capacity}} & \[C_y\] & places & \[C_{y_{se}} + C_{y_{st}}\] & Nombre de places assises et debout dans le véhicule. \\
-\hline
-\label{unit_capacity}
-\makecell[r]{Capacité d'unité \\ \textit{Unit capacity}} & \[C_{u}\] & places & \[C_{{u}_{se}} + C_{{u}_{st}}\] & Nombre de places assises et debout dans l'unité. \\
-\hline
-\label{vehicle_seated_capacity}
-\makecell[r]{Capacité de sièges par véhicule \\ \textit{Vehicle seated capacity}} & \[C_{y_{se}}\] & places & - & - \\
-\hline
-\label{vehicle_standees_capacity}
-\makecell[r]{Capacité de passagers debout par véhicule \\ \textit{Vehicle standees capacity}} & \[C_{y_{st}}\] & places & - & Attention de bien préciser l'espace disponible par place debout \(e\) entre \(0.15m^2\) (inconfortable) et \(0.25m^2\) (confortable). \\
-\hline
-\label{unit_seated_capacity}
-\makecell[r]{Capacité de sièges par unité \\ \textit{Unit seated capacity}} & \[C_{u_{se}}\] & places & \[n_y C_{{y}_{se}}\] si toutes les voitures de l'unité ont la même capacité de sièges. & - \\
-\hline
-\label{unit_standees_capacity}
-\makecell[r]{Capacité de passagers debout par unité \\ \textit{Unit standees capacity}} & \[C_{u_{st}}\] & places & \[n_y C_{{y}_{st}}\] si toutes les voitures de l'unité ont la même capacité de places debout. & - \\
+\makecell[r]{Floor area per standee} & \[e\] & \[m^2\] & - & Floor area allocated for each standing place (footprint). Comfortable, adequate circulation: \(0.25 m^2\); Uncomfortable, difficult circulation: \(0.15 m^2\) \\
 \hline
 \label{vehicle_boarding_door_channels}
-\makecell[r]{Nombre de voies d'embarquement par véhicule \\ \textit{Number of boarding door channels per vehicle}} & \[{n_{Bch}}_y\] & voies d'embarquement & - & Nombre de voies permettant l'embarquement d'une personne à la fois dans le véhicule. \\
+\makecell[r]{Number of boarding door channels per vehicle} & \[{n_{Bch}}y\] &boarding channels & - & Number of channels allowing one person at a time to board the vehicle. \\
```

Comment:
DO you need a space after the "&" ?
Also dropped the _ before the y

Reply:
fixed